From 7764aa910c3b3e5614801c72442e21fd3e8083db Mon Sep 17 00:00:00 2001
Message-Id: <7764aa910c3b3e5614801c72442e21fd3e8083db.1352118319.git.jdenemar@redhat.com>
From: Martin Kletzander <mkletzan@redhat.com>
Date: Tue, 30 Oct 2012 11:41:52 +0100
Subject: [PATCH] esx: Fix connection to ESX 5.1

https://bugzilla.redhat.com/show_bug.cgi?id=865670

After separating 5.x and 5.1 versions of ESX, we forgot to add 5.1
into the list of allowed connections, so connections to 5.1 fail since
v1.0.0-rc1-5-g1e7cd39
(cherry picked from commit bab7752c0c6e82139f704b83f381a4c34a7b0f39)
---
 src/esx/esx_driver.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/esx/esx_driver.c b/src/esx/esx_driver.c
index 8d13829..2aa6978 100644
--- a/src/esx/esx_driver.c
+++ b/src/esx/esx_driver.c
@@ -718,6 +718,7 @@ esxConnectToHost(virConnectPtr conn,
             priv->host->productVersion != esxVI_ProductVersion_ESX41 &&
             priv->host->productVersion != esxVI_ProductVersion_ESX4x &&
             priv->host->productVersion != esxVI_ProductVersion_ESX50 &&
+            priv->host->productVersion != esxVI_ProductVersion_ESX51 &&
             priv->host->productVersion != esxVI_ProductVersion_ESX5x) {
             virReportError(VIR_ERR_INTERNAL_ERROR,
                            _("%s is neither an ESX 3.5, 4.x nor 5.x host"),
@@ -847,6 +848,7 @@ esxConnectToVCenter(virConnectPtr conn,
         priv->vCenter->productVersion != esxVI_ProductVersion_VPX41 &&
         priv->vCenter->productVersion != esxVI_ProductVersion_VPX4x &&
         priv->vCenter->productVersion != esxVI_ProductVersion_VPX50 &&
+        priv->vCenter->productVersion != esxVI_ProductVersion_VPX51 &&
         priv->vCenter->productVersion != esxVI_ProductVersion_VPX5x) {
         virReportError(VIR_ERR_INTERNAL_ERROR,
                        _("%s is neither a vCenter 2.5, 4.x nor 5.x server"),
-- 
1.8.0

